libs/partners/nomic/langchain_nomic/embeddings.py

```diff
@@ -121,3 +124,27 @@ def embed_query(self, text: str) -> List[float]:
             texts=[text],
             task_type="search_query",
         )[0]
+
+
+class NomicMultimodalEmbeddings(NomicEmbeddings):
```

Comment:
might it be a simpler / more intuitive devx if NomicEmbeddings just implemented the multimodal functionality?

Reply:
updated!